runtime/queries/nix/injections.scm

```diff
@@ -129,3 +129,34 @@
   ])
   (#match? @_func "(^|\\.)writeRust(Bin)?$")
   (#set! injection.combined))
+
+;; (nixosTest) testScript
+((binding
+  ((attrpath) @_attr_name) 
+    (#eq? _attr_name "nodes")
+  )
+  (binding
+   ((attrpath) @_func_name) (#eq? @_func_name "testScript")
+   (_
+     (string_fragment) @injection.content (#set! injection.language "python")
+   ))
+   (#set! injection.combined))
```

Comment:
Are testScripts always in python? If yes then lgtm

Reply:
Yes, I'm matching 
```nix
{
 nodes = { ... }
 testScript = ''
 '';
}
```
where `testScript` should always be Python used to interact with virtual machines defined in `nodes`.

I just updated my local injections and they're no longer working, so I'll have to come back to this.
I see the directory structure has changed - maybe I'll have to update my local injections accordingly.